e2e/cypress/integration/team_settings/manage_members_spec.js

```diff
@@ -0,0 +1,188 @@
+
+                // # Click Main Menu
+                cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+                // # Click Manage Members
+                cy.get('#manageMembers').should('be.visible').click();
+
+                // # Open member dropdown
+                cy.get(`#teamMembersDropdown_${user.username}`).should('be.visible').click();
+
+                // # Click Make Team Admin
+                cy.get(`#teamMembersDropdown_${user.username} ~ div button:contains(Make Team Admin)`).should('be.visible').click();
+
+                // * Verify dropdown shows that user is now a Team Admin
+                cy.get(`#teamMembersDropdown_${user.username} span:contains(Team Admin)`).should('be.visible');
+            });
+        });
+    });
+
+    it('MM-T2335 Remove a team member and ensure they cannot rejoin if the team is not joinable', () => {
+        // # Make the test user a Team Admin
+        promoteToChannelOrTeamAdmin(testUser.id, testTeam.id, 'teams');
+
+        // # Create a new user
+        cy.apiCreateUser({prefix: 'nonAdminUser'}).then(({user}) => {
+            // # Add user to the team
+            cy.apiAddUserToTeam(testTeam.id, user.id).then(() => {
+                // # Create a second team
+                cy.apiCreateTeam('other-team', 'Other Team').then(({team: otherTeam}) => {
+                    // # Add user to the other team
+                    cy.apiAddUserToTeam(otherTeam.id, user.id).then(() => {
+                        // # Login as new team admin
+                        cy.apiLogin(testUser);
+
+                        // # Go to Town Square
+                        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+                        // # Click Main Menu
+                        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+                        // # Click Manage Members
+                        cy.get('#manageMembers').should('be.visible').click();
+
+                        // # Open member dropdown
+                        cy.get(`#teamMembersDropdown_${user.username}`).should('be.visible').click();
+
+                        // # Click Remove from Team
+                        cy.get(`#teamMembersDropdown_${user.username} ~ div button:contains(Remove from Team)`).should('be.visible').click();
+
+                        // * Verify teammate no longer appears
+                        cy.get(`#teamMembersDropdown_${user.username}`).should('not.be.visible');
+
+                        // # Login as non admin user
+                        cy.apiLogin(user);
+
+                        // # Go to team that they are still a member of
+                        cy.visit(`/${otherTeam.name}/channels/town-square`);
+
+                        // * Verify they are still a member
+                        cy.get(`#headerTeamName:contains(${otherTeam.display_name})`, {timeout: 45000}).should('be.visible');
```

Comment:
Please use available timeout constants. Could be one minute or feel free to add a new one.